moonwalker-migrations/process-queue.js

```diff
@@ -0,0 +1,22 @@
+const EthDeployer = require('moonwalker').default
+const HDWalletProvider = require('@truffle/hdwallet-provider')
+const config = require('./config')
+
+const wallet = new HDWalletProvider(config.mnemonic, config.url)
+
+async function consume() {
+  const q = await EthDeployer.getQueue()
+  const worker = new EthDeployer.Worker(
+    wallet, q, {
+      from: config.from,
+      gas: 6000000,
+      gasPrice: '45000000000' // 45 gwei
```

Comment:
i guess will need to pass more for the mainnet.

Reply:
done